### PR TITLE
set signing to auto for IOS debug builds [Xcode]

### LINKED ISSUE #15419: Automatic Certificate Signing at Xcode
When opening the project in Xcode we get build failures due to certificates not being valid for development and distribution.
They are solved when we select automatic signing in Xcode and delete the provisioned profiles from build settings.
It is cumbersome to do this every-time.

The goal here is to fix the problem to ensure that CCs can build the app with Xcode without these errors while making sure that the CI build does not fail.

### PR DESCRIPTION
**Pull request by siddarthkay**
fixes #15419

#### Platforms
- iOS

### Steps to test
This change should only affects the build process of IOS debug builds

status: ready

### CODE REVIEW

**Review by ibrkhalil — APPROVED** [review]

**Comment by siddarthkay** [comment]
@status-im/mobile-qa : safe enough to merge? This only affects debug IOS build settings

**Comment by pavloburykh** [comment]
> @status-im/mobile-qa : safe enough to merge? This only affects debug IOS build settings

@siddarthkay thanx for the PR. Ready to be merged.

**Review by jakubgs — COMMENTED** [review]
I'd appreciate if you waited for me to take a look when you change anything related to signing.

**Comment by siddarthkay** [comment]
@jakubgs : My bad, sorry this skipped my mind. Will wait for a go-ahead next time 👍🏻

File: ios/StatusIm.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -917,9 +913,8 @@
 				BUNDLE_ID_SUFFIX = "";
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = StatusIm/StatusIm.entitlements;
-				CODE_SIGN_IDENTITY = "iPhone Developer";
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
-				CODE_SIGN_STYLE = Manual;
+				CODE_SIGN_IDENTITY = "Apple Development";
+				CODE_SIGN_STYLE = Automatic;
```
Comment (jakubgs) [inline ios/StatusIm.xcodeproj/project.pbxproj:917]: The title says this is changing debug builds, but this is config for a release build.

Reverting in:

* https://github.com/status-im/status-mobile/pull/15559

**Pull request merged**